dashboard/test/models/user_test.rb

```diff
@@ -1563,6 +1563,18 @@ def track_progress(user_id, script_level, result, pairings: nil)
     assert_equal [], teacher.reload.permissions
   end
 
+  test 'assign_course_as_facilitator assigns course to facilitator' do
+    facilitator = create :facilitator
+    facilitator.course_as_facilitator = Pd::Workshop::COURSES.last
```

Comment:
I'm not able to get `assert_create` to work.

This test seems to assign the string value `Pd::Workshop::COURSE_CS_IN_A ` to `course_facilitator`:

``` ruby
  test 'assign_course_as_facilitator assigns course to facilitator' do
    facilitator = create :facilitator
    course_facilitator = assert_creates Pd::CourseFacilitator do
      facilitator.course_as_facilitator = Pd::Workshop::COURSE_CS_IN_A
    end
    assert_equal Pd::Workshop::COURSE_CS_IN_A, course_facilitator.course
    assert_equal facilitator, course_facilitator.facilitator
  end
```

So the test FAILs with the error:

```
NoMethodError:         NoMethodError: undefined method `course' for "CS in Algebra":String
            test/models/user_test.rb:1577:in `block in <class:UserTest>'
            test/testing/setup_all_and_teardown_all.rb:22:in `run'
```

I don't currently understand ruby enough to understate how `assert_create` (and `assert_difference`, which it invokes) work to debug this on my own.  I'm reverting back to the original test for now.